cristin-project/src/test/java/no/unit/nva/cristin/projects/create/CreateCristinProjectHandlerTest.java

```diff
@@ -407,6 +411,63 @@ void shouldRemoveFieldsNotSupportedByPostToUpstream() throws Exception {
         assertThat(keyword.getName(), equalTo(null));
     }
 
+    @Test
+    void shouldAddCreatorDataToInput() throws Exception {
+        var apiClient = new CreateCristinProjectApiClient(mockHttpClient);
+        apiClient = spy(apiClient);
+        handler = new CreateCristinProjectHandler(apiClient, environment);
+
+        var nvaProject = randomNvaProject();
+        nvaProject.setId(null);
+        mockUpstreamUsingRequest(nvaProject);
+        var input = inputWithClientIdentifiers(nvaProject);
+        handler.handleRequest(input, output, context);
+
+        var captor = ArgumentCaptor.forClass(NvaProject.class);
+        verify(apiClient).createProjectInCristin(captor.capture());
+        var capturedNvaProject = captor.getValue();
```

Comment:
I think using captors are over complicated and hide what we are doing, maybe there are some finer aspects that is lost to me?
Anyways, no change suggested

Reply:
It is more like I don't understand what is gained by doing it this way.
Why is this better than just receiving the dto directly?
What value is gained?

Reply:
A captor can read the parameter sent to the method that is not otherwise accessible by other means